### DISCUSSION TITLE
Issues getting template deployed.

### DISCUSSION THREAD

**Discussion by jlengelbrecht** [onedr0p/cluster-template#1551] [Q&A] [answered]
Hi there.

Following along with the documentation on the Kubernetes template but running into issues getting this deployed. 
I am attempting this install on 8 raspberry pis, but for some reason keep getting stuck when running `task talos:boostrap` command. 

After running the task I get a failure stating: 
```
unknown flag: --kubeconfig before the script quits with exit status 3. 
```
When I check the cluster state after running the task I can see all of my nodes but they are in a `NotReady` state 
On talos itself the controlplanes report:
```
 {"component": "controller-runtime", "controller","k8s.KubeletStaticPodController", "error": "error" refreshing pod status: error fetching pod status: Get \"https://127.0.0.1:10250/pods/?timeout=30s\":remote error: tls: internal error"}
```
```
pi_cluster on git main [?] via py v3.12.4 (.venv) 
x kubectl get nodes 
NAME      STATUS     ROLES           AGE   VERSION
master1   NotReady   control-plane   12m   v1.30.3
master2   NotReady   control-plane   12m   v1.30.3
master3   NotReady   control-plane   12m   v1.30.3
worker1   NotReady   <none>          12m   v1.30.3
worker2   NotReady   <none>          12m   v1.30.3
worker3   NotReady   <none>          12m   v1.30.3
worker4   NotReady   <none>          12m   v1.30.3
worker5   NotReady   <none>          12m   v1.30.3
```
Attached is a pastebin of my config.yaml hiding any sensitive info and the logs from my terminal.

[config.yaml](https://pastebin.com/kkLLFM92) 
[Terminal Logs](https://pastebin.com/hdFe2Xdg)

Some things about my setup: 

- Arch as my host system. Not running the devcontainer but actually installing dependencies onto my system.
- Installing on 8 raspberrypi 4's each with 8gb and 125tb SSD that's running talos with the following `schematic id: 376567988ad370138ad8b2698212367b8edcb69b5fd68c80be1f2ec7d603b4ba`
- Using a private repo inside of my own github org.
- DHCP reservation has been made for each node but not for the ingress_vip, bootstrap_controller_vip, or the gateway_vip.

Any help in understanding  why this keeps failing would be greatly appreciated. 
I feel I have exhausted all of resources at this point and am thoroughly stuck :hankey:

**Comment by onedr0p** [comment, marked as answer]
You likely don't have the most recent version of helmfile which means cilium and the other apps in that config file are not installed.

https://github.com/helmfile/helmfile

The one in the aur is out of date. 

https://aur.archlinux.org/packages/helmfile-bin

I went ahead and flagged it in the AUR as out of date.

**Reply by jfroy** [reply]
Package updated.

**Reply by jlengelbrecht** [reply]
This worked. Thank you.
